packages/babel-core/src/config/validation/options.js

```diff
@@ -216,6 +243,9 @@ export function validate(type: OptionsType, opts: {}): ValidatedOptions {
     if (type === "env" && key === "env") {
       throw new Error(`.${key} is not allowed inside another env block`);
     }
+    if (type === "env" && key === "override") {
+      throw new Error(`.${key} is not allowed inside an env block`);
+    }
```

Comment:
Should we also disable `override` in another `override` block?

Reply:
I would say yes.

Reply:
Good catch, definitely an oversight.